backend/routes/productRoutes.js

```diff
@@ -7,4 +7,20 @@ router.get('/', async (req, res) => {
     res.send(products);
 });
 
+router.post('/', (req, res) => {
+    // I pass the body of the request as argument of the Product constructor and assign to a variable
+    let product = new Product(req.body);
+
+    // I send the product to the db and once the sending is ok I redirect to the same endpoint like before 
+    product.save()
```

Comment:
Also if save has an async method that would be preferred instead of the promises it's a little cleaner to read

Reply:
Ok I'll do a try/catch if it's more readable.